with-dev-client/.expo-shared/assets.json

```diff
@@ -0,0 +1,4 @@
+{
+  "12bb71342c6255bbf50437ec8f4441c083f47cdb74bd89160c15e4f43e52a1cb": true,
+  "40b842e832070c58deac6aa9e08fa459302ee3f9da492c7e77d93d2fbf4a56fd": true
```

Comment:
Just for my knowledge ;) To which assets those strings refers?

Reply:
Haha. 😆 Actually it looks like it refers to none of them. This file must have come from the managed app template. I ran `npx expo-optimize` and it created a file with different hashes:
```json
{
  "5f4c0a732b6325bf4071d9124d2ae67e037cb24fcc9c482ef82bea742109a3b8": true,
  "24272cdaeff82cc5facdaccd982a6f05b60c4504704bbf94c19a6388659880bb": true,
  "74c64047eb557b1341bba7a2831eedde9ddb705e6451a9ad9f5552bf558f13de": true,
  "052227dc810848b3a2fc99161a392256ea9ef37da10b69b6630ccf6dbb5e2ca6": true
}
```


Expo CLI used to complain about unoptimized assets if this file didn't exist. AFAIK it doesn't complain anymore so we might as well delete it.